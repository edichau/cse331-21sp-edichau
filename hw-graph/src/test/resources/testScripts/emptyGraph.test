# Tests that a graph can be created

# Create a graph
CreateGraph graph1

# Print the nodes in the graph to verify it is empty
ListNodes graph1