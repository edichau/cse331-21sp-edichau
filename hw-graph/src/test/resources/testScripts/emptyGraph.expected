# Tests that a graph can be created

# Create a graph
created graph graph1

# Print the nodes in the graph to verify it is empty
graph1 contains: